testing/kuttl/e2e-other/standalone-pgadmin-user-management/07-assert.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+# When setup.py returns users in Json, the Role translation is 1 for Admin, 2 for User
+- script: |
+    pod_name=$(kubectl get pod -n "${NAMESPACE}" -l postgres-operator.crunchydata.com/pgadmin=pgadmin -o name)
+    secret_name=$(kubectl get secret -n "${NAMESPACE}" -l postgres-operator.crunchydata.com/pgadmin=pgadmin -o name)
+
+    users_in_pgadmin=$(kubectl exec -n "${NAMESPACE}" "${pod_name}" -- bash -c "python3 /usr/local/lib/python3.11/site-packages/pgadmin4/setup.py get-users --json")
+    
+    bob_role=$(jq '.[] | select(.username=="bob@example.com") | .role' <<< $users_in_pgadmin)
+    dave_role=$(jq '.[] | select(.username=="dave@example.com") | .role' <<< $users_in_pgadmin)
+    jimi_role=$(jq '.[] | select(.username=="jimi@example.com") | .role' <<< $users_in_pgadmin)
+
+    [ $bob_role = 1 ] && [ $dave_role = 1 ] && [ $jimi_role = 2 ] || exit 1
+
+    users_in_secret=$(kubectl get "${secret_name}" -n "${NAMESPACE}" -o 'go-template={{index .data "users.json" }}' | base64 -d)
+
+    $(jq '. == []' <<< $users_in_secret) || exit 1
```

Comment:
:thinking: Is this check just making sure an array is returned?

Reply:
It is ensuring that the `users_in_secret` is an empty array (because the users were deleted from the spec and should therefore be removed from `users.json` in the secret).

Reply:
This is mentioned in the README.md... Is that sufficient?